Rendering/WDE.MapSpawns/Rendering/SpawnContextMenu.cs

```diff
@@ -1,23 +1,111 @@
+using System;
 using System.Windows.Input;
 using Prism.Commands;
 using WDE.Common.Services;
+using WDE.Common.Services.MessageBox;
 using WDE.MapSpawns.Models;
 using WDE.MapSpawns.ViewModels;
+using WDE.SqlQueryGenerator;
+using WDE.DatabaseEditors.Services;
+using WDE.DatabaseEditors.Data.Interfaces;
 
 namespace WDE.MapSpawns.Rendering;
 
 public class SpawnContextMenu
 {
     private readonly ISpawnSelectionService spawnSelectionService;
-    
+    private readonly IDatabaseQueryExecutor mySqlExecutor;
+    private readonly ITableDefinitionProvider definitionProvider;
+    private readonly IMessageBoxService messageBoxService;
+
     private ICommand CopyGuidCommand { get; }
+    private ICommand CopyPositionXCommand { get; }
+    private ICommand CopyPositionYCommand { get; }
+    private ICommand CopyPositionZCommand { get; }
+    private ICommand CopyOrientationCommand { get; }
+    private ICommand UpdateValuesCommand { get; }
 
-    public SpawnContextMenu(ISpawnSelectionService spawnSelectionService,
-        IClipboardService clipboardService)
+    public SpawnContextMenu(
+        ISpawnSelectionService spawnSelectionService,
+        IClipboardService clipboardService,
+        ITableEditorPickerService tableEditorPickerService,
+        IDatabaseQueryExecutor mySqlExecutor,
```

Comment:
I think here I'd use `IMySqlExecutor` instead of `IDatabaseQueryExecutor`, those are bad names I can see now...
`IMySqlExecutor` - executes queries on the world database
`IMySqlHotfixExecutor` - executes queries on the hotfixes database
`IDatabaseQueryExecutor` - has additional parameter which determines if the query should be executed in the world database or hotfixes database.

I can see why you chose the `IDatabaseQueryExecutor` here, however the intention was to keep  `IDatabaseQueryExecutor` only for the `WDE.DatabaseEditors` project. While database structure varies between different wow core, I think we can safely assume that creatures/gameobjects tables will be always in the world database.